packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -473,6 +473,124 @@ describe("Presence", () => {
 					});
 				});
 			});
+
+			describe("attendees", () => {
+				const firstAttendeeSessionId = "sessionId-4";
+				const firstAttendeeConnectionId = "client4";
+				const secondAttendeeSessionId = "sessionId-5";
+				const secondAttendeeConnectionId = "client5";
+				let firstAttendeeSignal: ReturnType<typeof generateBasicClientJoin>;
+				let secondAttendeeSignal: ReturnType<typeof generateBasicClientJoin>;
+				let firstAttendeeSignalWithUpdate: ReturnType<typeof generateBasicClientJoin>;
+
+				beforeEach(() => {
+					// Ignore submitted signals
+					runtime.submitSignal = () => {};
+
+					firstAttendeeSignal = generateBasicClientJoin(clock.now - 50, {
+						averageLatency: 50,
+						clientSessionId: firstAttendeeSessionId,
+						clientConnectionId: firstAttendeeConnectionId,
+						updateProviders: ["client2"],
+					});
+
+					secondAttendeeSignal = generateBasicClientJoin(clock.now - 50, {
+						averageLatency: 50,
+						clientSessionId: secondAttendeeSessionId,
+						clientConnectionId: secondAttendeeConnectionId,
+						updateProviders: ["client4"],
+					});
+
+					firstAttendeeSignalWithUpdate = generateBasicClientJoin(clock.now - 50, {
+						averageLatency: 50,
+						clientSessionId: firstAttendeeSessionId,
+						clientConnectionId: firstAttendeeConnectionId,
```

Comment:
I now see that the current connection Id is sent in the helper. I updated the signal so there is no collision. I kept the connection ID the same since in the test case the client remains connected throughout.